scripts/release/asana-update-tasks.js

```diff
@@ -44,13 +42,6 @@ const asanaUpdateTasks = async () => {
         const taskDescriptionSubstitutions = [[/\[\[pr_url]]/, prLink]];
 
         let extraContent = '';
```

Comment:
This could be removed now - if we don't use` [[extra_content]]` for anything else then it could be removed entirely. Happy to take suggestions on the correct approach here.